lib/ansible/modules/cloud/vmware/vmware_category.py

```diff
@@ -68,6 +68,27 @@
       default: 'present'
       choices: [ 'present', 'absent' ]
       type: str
+    associable_object_types:
+      description:
+      - List of object types that can be associated with the given category.
+      choices:
+      - All objects
+      - Folder
+      - Cluster
+      - Datacenter
+      - Datastore
+      - Datastore Cluster
+      - Distributed Port Group
+      - Distributed Switch
+      - Host
+      - Content Library
+      - Library item
+      - Network
+      - Resource Pool
+      - vApp
+      - Virtual Machine
+      version_added: '2.10'
+      type: list
```

Comment:
```suggestion
      type: list
      elements: str
```
and would be cool to see the choices in alphabetical order